windows/build-artifacts.ps1

```diff
@@ -41,4 +41,4 @@ Invoke-Call -ScriptBlock { cbindgen --crate data-pipeline-ffi --config data-pipe
 Invoke-Call -ScriptBlock { cbindgen --crate datadog-crashtracker-ffi --config crashtracker-ffi/cbindgen.toml --output $output_dir"\crashtracker.h" }
 Invoke-Call -ScriptBlock { .\target\release\dedup_headers $output_dir"\common.h"  $output_dir"\profiling.h" $output_dir"\telemetry.h" $output_dir"\data-pipeline.h" $output_dir"\crashtracker.h"}
 
-Write-Host "Build finished"  -ForegroundColor Magenta
\ No newline at end of file
+Write-Host "Build finished"  -ForegroundColor Magenta
```

Comment:
> /question
> 
> I thought we are going to replace `cargo build` from L25-28 with the new builder crate version?

For now, on the gitlab builder, if we target x86, the build fails. so for now we keep this as it is and when the build is fix this will be update.

Reply:
> I see you mentioned that we are going to do it later in the description. However, we must add some CI step for building for Windows to keep the feedback loop short.

Not sure to understand what do you mean by `we must add some CI steps`.  On each PR, this script is used to build on Windows. So there are already CI steps.